src/wp-includes/rest-api/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -36,6 +36,20 @@ public function __construct() {
 		$this->meta = new WP_REST_Comment_Meta_Fields();
 	}
 
+	/**
+	 * Send a notification to the post author when a new note is added via the REST API.
+	 *
+	 * @since 6.9.0
+	 *
+	 * @param WP_Comment $comment The comment object.
+	 */
+	public static function wp_new_comment_via_rest_notify_postauthor( $comment ) {
```

Comment:
Sorry, I wasn't able to find reasoning for moving this method here. So what was it?

IMO, seems odd and unrelated to have action callback in REST controller. It kind of goes against the idea of separation concerns. @TimothyBJacobs, what do you think?

Reply:
I had it in comments.php previously, but noticed some unit test failure because the hook was being fired in the test with a null comment. I can move it back and add a guard for this instead. I wasn't really sure about the best location for this code.

Reply:
Let’s move it before RC so we don’t introduce accidental public function that we have to deprecate in next release.